run_entrypoints/run_entrypoint_europe.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime, timedelta
+
+import numpy as np
+from seismostats.io.client import FDSNWSEventClient
+from shapely.geometry import Polygon
+from shapely.wkt import dumps
+
+from etas.oef import entrypoint_europe
+
+
+def main():
+    """
+    Requires to install the package with 'hermes' extras.
+    pip install -e .[hermes]
+
+    Makes use of the standardized interface to run the model.
+
+    More information under https://gitlab.seismo.ethz.ch/indu/hermes-model
+    """
+
+    format = '%Y-%m-%d %H:%M:%S'
+    auxiliary_start = datetime.strptime("2015-01-01 00:00:00", format)
+    timewindow_start = datetime.strptime("2015-01-01 00:00:00", format)
+    timewindow_end = datetime.strptime("2023-02-06 00:00:00", format)
+    # timewindow_end at later points causes an error in to_quakeml()
+
+    min_longitude = -38
+    max_longitude = 48
+    min_latitude = 25
+    max_latitude = 73
+
+    min_magnitude = 4.5
+    url = 'https://www.seismicportal.eu/fdsnws/event/1/query'
+    client = FDSNWSEventClient(url)
+    catalog = client.get_events(
+        start_time=timewindow_start,
+        end_time=timewindow_end,
+        min_magnitude=min_magnitude,
+        min_longitude=min_longitude,
+        max_longitude=max_longitude,
+        min_latitude=min_latitude,
+        max_latitude=max_latitude)
+
+    polygon = Polygon(np.load('../etas/oef/data/europe_shape_r.npy'))
+
+    forecast_duration = 30  # days
+
+    model_input = {
+        'forecast_start': timewindow_end,
+        'forecast_end': timewindow_end + timedelta(days=forecast_duration),
+        'bounding_polygon': dumps(polygon),
+        'depth_min': 0,
+        'depth_max': 1,                         # always in WGS84
+        'seismicity_observation': catalog.to_quakeml(),
+        'model_parameters': {
+            "theta_0": {},
+            "mc": 4.6,
+            "delta_m": 0.2,
+            "coppersmith_multiplier": 100,
+            "earth_radius": 6.3781e3,
+            "auxiliary_start": auxiliary_start,
+            "timewindow_start": timewindow_start,
+            "n_simulations": 100,
+            "fn_parameters": '../etas/oef/data/europe_parameters.json'
```

Comment:
sorry, but I have the same question again:
Is this: 
- a property of the model -> it should not be an input
- configuration parameters -> it should not be "hardcoded" in a file
- part of the model you want to be able to switch out -> then okay, it can stay here

Reply:
They are config parameters, used to initialize the `ETASParameterCalculation` object which is then used to make a simulation object which runs simulation as a method. Basically a lot of these inconsistencies stem from the fact that that config file is the inversion output (parameters are trained 1980-2015, etc.), then we instantiate an object, but 'run over' some of its properties as we want to simulate a continuation of a catalog that's different than the training catalog. It's a bit hacky altogether, sorry... maybe it's better if we don't define the object at all using the `.load_calculation()` method, but just pass all the parameters and initialize the class normally. It's all a remnant of how I do it when running things usually...

Reply:
yes, that would be nice if you can improve this. 

The configurations are the dials on the machine. 
1. Everything you want to be able to change, before pressing the "start" button, should be a dial. 
2. If you do not need to be able to change it, dont make a dial for it.
3. If you want to change something which has no dial, you will need to open the machine and change it internally.
4. If you need your machine to do too many different things, maybe you should just build 2 separate machines.

so, practically, for each above point:
1. think about what should be in the configuration.
2. everything you don't need in the configuration, handle it in the entrypoint directly.
3. meaning you would have to change something in the model itself, push it to git, reinstall it in hermes.
4. meaning you'd create another entrypoint and have a new "model".